src/services/namingService.ts

```diff
@@ -85,8 +84,8 @@ export class AzureNamingService {
       maxLength -= timestamp.length + suffix.length;;
 
       const name = (deploymentName) ? deploymentName.substr(0, maxLength)
-        : [ AzureNamingService.getSafeResourceName(config, maxLength), suffix ].join("-");
-      return [ name, timestamp ].join("-");
+        : [AzureNamingService.getSafeResourceName(config, maxLength), suffix].join("-");
```

Comment:
this need to now include the service name otherwise this change will break deployment name

Reply:
Ah - didn't know `deployment name` was generated from this also.  Will address.